src/sql/workbench/common/editor/query/queryEditorInput.ts

```diff
@@ -346,6 +321,26 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
 			}
 		}
 		this._onDidChangeLabel.fire();
+
+		this.contextualizeEditorForCopilot();
```

Comment:
Add a comment here explaining that you're purposely not awaiting (I'd even suggest adding a `void` before this to clearly show this is intentional)